locales.software/1038-hu.cfos.txt

```diff
@@ -91,14 +91,14 @@ pnet_conn_local_ip = "Helyi IP cím %?";
 pnet_conn_public_ip = "Nyílvános IP cím %?";
 pnet_pubdir_header = "Nyílvános könyvtár";
 pnet_pubdir_subheader = "Az a könyvtár, amelyik tartalmazza a letölthető fájlokat.";
-pnet_pubdir_info = "Kérlek, add meg azt a mappa helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
+pnet_pubdir_info = "Kérlek, add meg annak a mappának a helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
 smtp_header = "SMTP-szerver";
 smtp_subheader = "SMTP-szerver beállítása.";
 dynip_header = "Dinamikus IP-szerver";
 dynip_subheader = "Dinamikus IP-szerver beállítása";
 dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
 dlg_autostart = "%p indítása a Windows-zal együtt";
-dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
+dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
```

Comment:
I'm not sure that `%p` should be placed in `()`. Is there any particular reason behind it?